src/Microsoft.AspNetCore.Mvc.Testing/build/netstandard2.0/Microsoft.AspNetCore.Mvc.Testing.targets

```diff
@@ -15,11 +15,38 @@
     <PreserveCompilationContext>true</PreserveCompilationContext>
   </PropertyGroup>
 
-  <Target Name="CopyAditionalFiles" AfterTargets="Build" Condition="'$(TargetFramework)'!=''">
+    <Target Name="_ResolveMvcTestProjectReferences" DependsOnTargets="ResolveReferences">
+      <ItemGroup>
+        <_ContentRootProjectReferences Include="@(ReferencePath)" Condition="'%(ReferencePath.ReferenceSourceTarget)' == 'ProjectReference'" />
+      </ItemGroup>
+    </Target>
+
+  <Target Name="_AddContentRootForProjectReferences" BeforeTargets="BeforeCompile" DependsOnTargets="_ResolveMvcTestProjectReferences">
+    <ItemGroup>
+      <WebApplicationFactoryContentRootAttribute 
+        Include="%(_ContentRootProjectReferences.Identity)"
+        AssemblyName="%(_ContentRootProjectReferences.FusionName)"
+        ContentRootPath="$([System.IO.Path]::GetDirectoryName(%(_ContentRootProjectReferences.MSBuildSourceProjectFile)))"
+        ContentRootTest="$([System.IO.Path]::GetFileName(%(_ContentRootProjectReferences.MSBuildSourceProjectFile)))"
+        Priority="0" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <AssemblyAttribute Include="Microsoft.AspNetCore.Mvc.Testing.WebApplicationFactoryContentRootAttribute">
+        <_Parameter1>%(WebApplicationFactoryContentRootAttribute.AssemblyName)</_Parameter1>
+        <_Parameter2>%(WebApplicationFactoryContentRootAttribute.ContentRootPath)</_Parameter2>
+        <_Parameter3>%(WebApplicationFactoryContentRootAttribute.ContentRootTest)</_Parameter3>
+        <_Parameter4>%(WebApplicationFactoryContentRootAttribute.Priority)</_Parameter4>
+      </AssemblyAttribute>
+    </ItemGroup>
+  </Target>
+
+  <Target Name="CopyAditionalFiles" AfterTargets="Build;_ResolveMvcTestProjectReferences" Condition="'$(TargetFramework)'!=''">
     <ItemGroup>
-      <DepsFilePaths Include="$([System.IO.Path]::ChangeExtension('%(_ResolvedProjectReferencePaths.FullPath)', '.deps.json'))" />
+      <DepsFilePaths Include="$([System.IO.Path]::ChangeExtension('%(_ContentRootProjectReferences.ResolvedFrom)', '.deps.json'))" />
```

Comment:
Same here. Make sure to check that _ContentRootProjectReferences items actually exist.